SimG4Core/Notification/interface/TrackWithHistory.h

```diff
@@ -88,6 +93,8 @@ class TrackWithHistory {
   bool storeTrack_{false};
   bool saved_{false};
   bool crossedBoundary_{false};
+  bool isFromBackScattering_{false};
```

Comment:
these additions seem to duplicate information already available in ```TrackInformation```, where are they needed because they cannot be replaced by ```TrackInformation```?

Reply:
answering to myself, apparently the ```SimTrackManager``` is based on ```TrackWithHistory``` at present